package.json

```diff
@@ -13,6 +13,7 @@
     "@types/react-dom": "^17.0.2",
     "@types/react-router-dom": "^5.1.6",
     "@types/styled-components": "^5.1.4",
+    "date-fns": "^2.19.0",
```

Comment:
`date-fns` is a pretty big package with an unpacked size of ~5.5MB (!), and even the minified version is still around 85KB, see https://bundlephobia.com/result?p=date-fns@2.19.0

The 85KB on its own aren't too crazy, but this will quickly add up and slow down page load time. Since we are using just a very small portion of the export, we might be able to slim it down a bit with tree-shaking (i.e. only bundling the functions that we actually use and "shaking" every thing else off), but format alone is also 6KB, **after** compression (gzip).

This was just for some context, but I was wondering if we couldn't juse use `toLocaleDateString()` for now, unless we need more complex functionality. Alternatively we could be using `dayjs`, which also provides other functionality, but the entire lib is half as big as just the `format()` export of `date-fns`.